index-pl.html

```diff
@@ -206,61 +206,61 @@ <h2>Why Masks Protect Us Twice</h2>
 
   <div class="container">
 
-    <h2>From People to The Population</h2>
+    <h2>Od Jednostek do Populacji</h2>
 
     <p>
-      So far, we've only looked at disease transmission between two people.
-      How do we go from here to understanding disease transmission in the <i>entire population</i>?
+      Do tej pory, przyglądaliśmy się jedynie rozprzestrzenianiu się choroby pomiedzy dwojgiem ludzi.
+      Jak możemy przełożyć to na zrozumienie jak działa rozprzestrzenianie sie wirusa <i>w całej populacji</i>?
     </p> 
 
-    <p>Well, in the extreme limits, this is straightforward.</p> 
+    <p>W przypadkach skrajnych jest to dość proste.</p> 
 
     <p>
-      For example, if <b>nobody</b> wore a mask, then whenever two people meet, the chance that <span class="red">neither wear a mask</span> is 100%.
+      Na przykład, jeśli założymy, że <b>nikt</b> nie nosi maski, wtedy zawsze gdy dwoje ludzi się spotyka, szansa, <span class="red">że żadne z nich nie ma maski,</span> wynosi 100%.
     </p>
 
     <network :maskusage="0"></network>
 
     <p>
-      So we'd only have to consider the <span class="red">first route</span> of disease transmission, and the population would see <span class="red">no drop</span> in disease transmission.
+      W takiej sytuacji wzięliśmy pod uwagę tylko <span class="red">pierwszą drogę</span> rozprzestrzeniania się choroby i w populacji nie zaobserwowalibysmy <span class="red">żadnego spadku</span> w roznoszeniu się choroby.
     </p>
 
     <p>
-      At the other extreme, if <b>everyone</b> wore a mask, then whenever two people meet, the chance that they <span class="yellow">both wear masks</span> is 100%.
+      W przypadku drugiej skrajności, gdy założymy, że <b>wszyscy</b> noszą maski, w przypadku spotkania dwojga ludzi, szansa, że <span class="yellow">oboje mają założone maski,</span> wynosi 100%.
     </p>
 
     <network :maskusage="1"></network>
 
     <p>
-      In this case, we’d only have to consider the <span class="yellow">last route</span> of disease transmission.
-      Assuming masks are 50% effective in each direction, the population would see a <span class="yellow">75% drop</span> in disease transmission.
+      W tym przypadku więliśmy pod uwagę jedynie <span class="yellow">ostatnią drogę</span> rozprzestrzeniania się choroby.
+      Zakładając, że maski mają 50-procentową skutechność w każdym kierunku, w populacji zaobserwoalibyśmy<span class="yellow">75-procentowy spadek</span> w roznoszeniu choroby.
     </p>
 
     <p>
-      So when <i>everyone</i> wears a mask (or when <i>no one</i> wears one), it's straightforward to calculate the drop in disease transmission in the population, because there's only one route involved.
+      Tak więc, gdy <i>wszyscy</i> noszą maski (lub gdy <i>nikt</i> ich nie nosi), łatwo jest obliczyć spadek w transmisji choroby pośród populacji, ponieważ tylko jedena ze skrajności jest uwzględniona.
```

Comment:
"ponieważ tylko jedena ze skrajności jest uwzględniona"

W oryginale chodzi o to, że tylko jeden sposób/droga roznoszenia choroby jest używana w obliczeniach. Myślę, że tłumaczenie powinno być czymś w stylu "ponieważ tylko jedna droga roznoszenia choroby jest używana w obliczeniach" : )

Reply:
Hmm, no tak: jedna droga jest używana w obliczeniach, ale jest to 'skrajny przypadek': kiedy wszyscy noszą maski. Druga skrajność to gdy nikt nie nosi masek.
Nie za dobrze się tłumaczy: when only one route is involved. 
Nie podoba mi się moja wersja ale twojej też czegoś brakuje. :) 
@monika-brandys sugestie?

Reply:
To może "ponieważ w tym skrajnym przypadku tylko jedna droga roznoszenia choroby jest używana w obliczeniach"? Co myślisz?

Reply:
'ponieważ jest to przypadek skrajny i w obliczeniach uwzględniamy tylko tą jedną drogę roznoszenia choroby' 
? 
:)

Reply:
Świetne, może być!